src/app/shared/enums/crud-actions.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum eCRUDActions {
```

Comment:
this enum name should have capital E as convention. Just like we have in product i.e IProduct. The name should be like this `ECrudActions`. Naming should be consistent throughout the application.